sycl/tools/sycl-ls/sycl-ls.cpp

```diff
@@ -154,7 +159,8 @@ static void printDeviceInfo(const device &Device, bool Verbose,
           Device.get_info<sycl::ext::intel::info::device::device_id>();
       std::cout << Prepend << "DeviceID          : " << DeviceID << std::endl;
     } else {
-      std::cout << Prepend << "DeviceID          : " << "UNKNOWN" << std::endl;
+      std::cout << Prepend << "DeviceID          : "
+                << "UNKNOWN" << std::endl;
```

Comment:
Can you drop this and the next unrelated change please?

Reply:
done